src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -203,6 +207,96 @@ public void writeVariableExpr_scopedStaticFinalDecl() {
     assertEquals(writerVisitor.write(), "public static final boolean x");
   }
 
+  @Test  
+  public void writeNormalBlockComment() {
+    String content = "this is a test comment";
+    BlockComment blockComment = BlockComment.builder().setComment(content).build();
+    String expected = "/** this is a test comment */\n";
+    String formattedComment = blockComment.accept(writerVisitor);
+    assertThat(formattedComment).isEqualTo(expected);
+  }
+
+  @Test
+  public void writeNormalLineComment() {
+    String content = "this is a test comment";
+    LineComment lineComment = LineComment.builder().setComment(content).build();
+    String expected = "// this is a test comment\n";
+    String formattedComment = lineComment.accept(writerVisitor);
+    assertThat(formattedComment).isEqualTo(expected);
+  }
+
+  @Test
+  public void writeLongLineComment() {
+    String content =
+        "this is a long test comment with so many words, hello world, hello again, hello for 3 times, blah, blah!";
+    LineComment lineComment = LineComment.builder().setComment(content).build();
+    String expected =
+        "// this is a long test comment with so many words, hello world, hello again, hello for 3 times,\n// blah, blah!\n";
+    String formattedComment = lineComment.accept(writerVisitor);
+    assertThat(formattedComment).isEqualTo(expected);
+  }
+
+  @Test
+  public void writeNormalJavaDocComment() {
+    String content = "this is a test comment";
+    String deprecatedText = "Use the {@link ArchivedBookName} class instead.";
+    String paramName = "shelfName";
+    String paramDescription =  "The name of the shelf where books are published to.";
+    String paragraph1 = "This class provides the ability to make remote calls to the backing service through method calls that map to API methods. Sample code to get started:";
+    String paragraph2 = "The surface of this class includes several types of Java methods for each of the API's methods:";
+
+    TryCatchStatement tryCatch =
+        TryCatchStatement.builder()
+            .setTryResourceExpr(createAssignmentExpr("condition", "false", TypeNode.BOOLEAN))
+            .setTryBody(
+                Arrays.asList(ExprStatement.withExpr(createAssignmentExpr("x", "3", TypeNode.INT))))
+            .setIsSampleCode(true)
+            .build();
+
+    tryCatch.accept(writerVisitor);
+    String sampleCode = writerVisitor.write();
+    List<String> htmlList = Arrays.asList("A flattened method.", " A request object method.", "A callable method.");
```

Comment:
Nit: As with the `paragraph` comment in the last round, this is simply a list and is not HTML-specific.

Reply:
Sounds reasonable to me, done.